bigbluebutton-html5/imports/ui/components/dropdown/list/styles.scss

```diff
@@ -8,6 +8,8 @@
   text-align: left;
   color: $color-gray-dark;
   display: flex;
+  overflow-wrap: break-word;
+  white-space: pre-line;
 
```

Comment:
Can a set width be added to this line? The items in the list are creating a new line when not needed. It would be better to have a specified width to be reached before a new line is added.
![image](https://user-images.githubusercontent.com/32152296/42471879-5a1c142a-838d-11e8-9da3-3273b23c6966.png)
![image](https://user-images.githubusercontent.com/32152296/42471888-67280c78-838d-11e8-8b95-0a29c8d23ed0.png)